dialplan/asterisk/extensions_lib_user.conf

```diff
@@ -112,7 +112,9 @@ same  =                 n(busy),Busy()
 same  =                 n,Return()
 
 exten = forward_voicemail,1,NoOp()
+same  =   n,CELGenUserEvent(XIVO_INCALL,${WAZO_TENANT_UUID})
```

Comment:
You should not add the event `XIVO_INCALL` here since it's going to change internal calls into incoming calls.

Here's what will happen in wazo-call-logd when it gets interpreted

```
    def interpret_xivo_incall(self, cel, call):
        call.direction = 'inbound'
        extra = extract_cel_extra(cel.extra)
        if not extra:
            return call

        call.set_tenant_uuid(extra['extra'])
        return call
```
You could add the tenant_uuid to the `WAZO_USER_MISSED_CALL` call instead.

Reply:
It has been fixed.